tests/test_amend.py

```diff
@@ -554,5 +554,98 @@ def test_error_messages_not_a_folder(self) -> None:
             )
 
 
+class TestDeleteUnknownComponent(AmendTestCase):
+    def test_delete_unknown_component(self) -> None:
+        licenses = [
+            {"license": {"name": "unknown.something"}},
+            {"license": {"name": "whateverUnknown"}},
+            {"license": {"name": "unKnowN etc"}},
+            {"license": {"name": "AunknowNM", "text": ""}},
+            {"license": {"name": "unknown", "text": "license text"}},
+            {"license": {"name": "some license", "text": "license text"}},
+            {"license": {"name": "some license"}},
+            {"license": {"name": ""}},
+        ]
+        component = {"licenses": copy.deepcopy(licenses)}
+        process_license.delete_unknown_license(component)
+        self.assertEqual(component["licenses"], licenses[4:])
+
+    def test_amend_delete_unknown_license(self) -> None:
+        sbom = {
```

Comment:
Please add a test for the simple case that only one license with e.g. `"name": "Unknown - See URL"`. This case does not work for me and is the one we faced recurrently.

Reply:
Added.